alacritty/src/display/window.rs

```diff
@@ -496,3 +507,33 @@ fn use_srgb_color_space(window: &WinitWindow) {
         let _: () = msg_send![raw_window, setColorSpace: NSColorSpace::sRGBColorSpace(nil)];
     }
 }
+
+#[cfg(target_os = "macos")]
+fn set_background_opacity(window: &WinitWindow, opacity: CGFloat) {
```

Comment:
These macOS-specific functions should indicate their macOS-specificity. They're also lacking function documentation.